example/apps/app-a/set-a.edn

```diff
@@ -0,0 +1,158 @@
+{:description
+ "Example project with internal and external dependencies.",
+ :compile-path
+ "/Users/stephen.gerkin/amperity/lein-monolith/example/apps/app-a/target/compiled",
+ :deploy-repositories
+ [["clojars"
+   {:url "https://repo.clojars.org/",
+    :password :gpg,
+    :username :gpg}]],
+ :group "lein-monolith.example",
+ :injections
+ [(require 'pjstadig.humane-test-output)
+  (pjstadig.humane-test-output/activate!)],
+ :test-refresh
+ {:notify-command
+  ["/opt/homebrew/bin/terminal-notifier"
+   "-group"
+   "lein-test-refresh"
+   "-title"
+   "lein test-refresh"
+   "-message"],
+  :changes-only true},
+ :monolith/dependency-set :set-a,
+ :deployable true,
+ :resource-paths
+ ("/Users/stephen.gerkin/amperity/lein-monolith/example/apps/app-a/dev-resources"
+  "/Users/stephen.gerkin/amperity/lein-monolith/example/apps/app-a/resources"),
+ :uberjar-merge-with
+ {"META-INF/plexus/components.xml" leiningen.uberjar/components-merger,
+  "data_readers.clj" leiningen.uberjar/clj-map-merger,
+  #"META-INF/services/.*"
+  [clojure.core/slurp
+   (fn*
+    [p1__7803__7805__auto__ p2__7804__7806__auto__]
+    (clojure.core/str
```

Comment:
Ah, this is a gensym argument symbol and is not going to be stable. More broadly, a bunch of these are leiningen defaults or appear to be from your personal leiningen user profile. This seems like something we shouldn't be checking in and testing against.

Reply:
this should be removed from the PR, I committed by mistake